### ISSUE TITLE
[Visual requirement – Make Code – Home Page] Tooltip is not defined for visually appearing (See More & See Previous) icon controls on ‘Home Page’.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3521]
**User Experience:**
User would find difficulty to identify the purpose of icon control if tooltip is not defined for visual icon and also user won't be able to perform functionality over that icon control.
 
**Repro-Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Verify tooltip of visual icon (See More & See Previous) controls.
 
**Actual Result:**
Tooltip is not defined for visually appearing (See More & See Previous) icon controls on ‘Home Page’, due to this user is unable to identify the purpose of icon control and also won't be able to take action on such controls.

**Expected Result:**
Tooltip should be defined as (See More & See Previous) for visually appearing icon control (</>) on ‘Home Page’, so that user could easily identify the purpose of icon control and perform functionality over that control.
 
**Note:** 
1. Same issue exists on ‘Chrome’ &  Firefox.
2. Same issue exists on 'Help' icon control present on 'Extensions' screen.

**WCAG Reference:**
[WCAG 1.3.3 - Sensory Characteristics](https://www.w3.org/WAI/WCAG21/Understanding/sensory-characteristics.html})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG1.3.3, #Win10-Edge(Chromium)

![WCAG1 3 3_Tooltip is not defined](https://user-images.githubusercontent.com/72871089/97665216-e68caa80-1aa0-11eb-9d7d-2cf33fe130d3.jpg)

**Comment by jwunderl** [comment]
should have title in addition to aria label
